dev/ansible/01_generate_secrets_vault.yml

```diff
@@ -1,9 +1,11 @@
+- name: Set vault token
+  shell: export VAULT_TOKEN=$vault_master_token
+
 - name: Generate Secrets with Vault
   shell: |
       vault kv put secret/postgres \
       username=$(vault write -field='value' gen/password length=12 symbols=0) \
       password=$(vault write -field='value' gen/password length=12 symbols=0)
   run_once: true
   environment:
-      VAULT_ADDR: "http://127.0.0.1:8200"
-      VAULT_TOKEN: $vault_master_token
```

Comment:
you might need to make lookup here is example https://github.com/fredrikhgrelland/vagrant-hashistack/blob/master/ansible/tests/enterprise/vault_namespaces_test.yml#L16

Reply:
This is outdated - created a new task instead that exports `VAULT_TOKEN`. But should probably use lookup :)